web/docs/deployment/deployment-methods/self-hosted.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Self-Hosted
+---
+
+If you have your own server or you rent out a server, you can self-host your Wasp apps. This deployment option has the benefit of giving you full control over your apps and its data. Also, it can be more cost-effective than using a cloud provider since you can deploy multiple apps on a single server.
+
+## What you'll need
+
+To successfully self-host your Wasp app, you need to have the following:
+- A server with a public IP address
+  - There are many cloud providers you can use to rent a server. Some popular ones are [AWS](https://aws.amazon.com/ec2/), [DigitalOcean](https://www.digitalocean.com/), [OVH](https://www.ovhcloud.com/en/vps/), and [Hetzner](https://www.hetzner.com/cloud/).
+- A domain name (we'll need this to have HTTPS)
+
+## Self-hosting steps
+
+To self-host your Wasp app, you need to follow these general steps:
+1. Get your app's code on the server. How you do this depends on your setup, we'll explore a few methods in the next section.
+2. Set up [Docker](https://docs.docker.com/engine/install/) on the server. This is needed to run the server and the database.
```

Comment:
I wouldn't recommend _not using Docker_ since our server Dockerfile handles database migrations and moves things around. I feel like this is the best way for our users to deploy the server app. Experienced users already know this and they are the ones that might go for some custom setup.